src/landscape.cpp

```diff
@@ -1368,26 +1368,26 @@ static std::tuple<bool, bool> FlowRiver(TileIndex spring, TileIndex begin, uint
 		/* Maybe we can make a lake. Find the Nth of the considered tiles. */
 		std::set<TileIndex>::const_iterator cit = marks.cbegin();
 		std::advance(cit, RandomRange(count - 1));
-		TileIndex lakeCenter = *cit;
+		TileIndex lake_center = *cit;
```

Comment:
As an English-language codebase, should we aim for `lake_centre`?

Reply:
sure